wordpress/wp-content/themes/headless/functions.php

```diff
@@ -73,6 +72,7 @@ function bubs_theme_options($wp_customize)
 // Security Settings
 // include_once 'setup/helpers/google-login-force.php';
 // include_once 'setup/helpers/password-rotation-wpe.php';
+include_once 'setup/helpers/google-login-cookies.php';
```

Comment:
this file is `cookies.php`, but the file name below is `cookie.php` (not plural)